test/message/test_runner_no_refs.js

```diff
@@ -0,0 +1,14 @@
+// Flags: --no-warnings
+'use strict';
+require('../common');
+const test = require('test_runner');
+
+// When run alone, the test() below does not keep the event loop alive. This
+// test verifies that the test harness keeps the event loop alive anyway.
```

Comment:
Not blocking, but I'm not convinced it should be the default behavior. For example, the following would keep the process alive indefinitely:

```js
test('never resolving promise', () => new Promise(() => {}));
```

This is not how top-level await works (the process would exit immediately with error code 13), not how Deno handles it:

```ts
Deno.test({
    name:'never resolving promise',
    fn: () => new Promise(() => {}),
});
```
```console
$ deno test denotest.ts
Check file://…/denotest.ts
running 1 test from file://…/denotest.ts
test never resolving promise ...
test result: FAILED. 0 passed; 0 failed; 0 ignored; 0 measured; 0 filtered out (28ms)

error: Promise resolution is still pending but the event loop has already resolved.
```

Reply:
:+1: I'll go with that approach. I didn't like the extra keep alive interval.